src/common/hooks/useFeaturedPlays.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { submit } from "common/services/request";
+import { FetchPlaysFilter } from "common/services/request/query/fetch-plays-filter";
+
```

Comment:
Provide a description of what is this hook supposed to do.

Reply:
this hooks is grabbing all the featured play. will add description.

Reply:
yes thats a good idea also and i think we should have a discussion in it also to extract all the hooks from different folder and put in one root level hook folder.